src/utils/createErrorResponse.ts

```diff
@@ -1,13 +1,59 @@
-export function createErrorResponse(error: string | Error | unknown) {
-  let reason
+export interface Response {
+  v: '2'
```

Comment:
What is the v2?

Reply:
`v` field is used to separate API v1 and API v2 responses by the Pro Agent.